lib/ts/recipe/passwordless/recipe.tsx

```diff
@@ -103,36 +102,39 @@ export default class Passwordless extends AuthRecipe<
     static init(
         config: UserInput
     ): RecipeInitResult<GetRedirectionURLContext, PreAndPostAPIHookAction, OnHandleEventContext, NormalisedConfig> {
-        const normalisedConfig = normalisePasswordlessConfig(config);
-
         return {
             recipeID: Passwordless.RECIPE_ID,
             authReact: (
-                appInfo: NormalisedAppInfo
+                appInfo
             ): RecipeModule<
                 GetRedirectionURLContext,
                 PreAndPostAPIHookAction,
                 OnHandleEventContext,
                 NormalisedConfig
             > => {
+                const normalisedConfig = normalisePasswordlessConfig(config);
```

Comment:
These recipe changes can be reverted entirely; they do not actually change anything but make us call the normalisation function twice.